README.rst

```diff
@@ -1,18 +1,16 @@
-Part of `edX code`__.
-
-__ http://code.edx.org/
-
-edx ORA (Open Response Assessor)
-=======================
+[ARCHIVED] edx ORA (Open Response Assessor)
+===========================================
 
 The ORA will take a submission from an xqueue installation, pass it through machine learning grading, peer grading, and staff grading as appropriate, and return a result to LMS.  This is to be used with the edx-platform and xqueue.  It allows for the assessment of open response problems on the edx platform.
 
 DEPRECATED
 ------------------------
 
-The edX team has officially transitioned all support for the Open Response Assessor (ORA) project to its second iteration (ORA2). The original ORA project will remain open on github, but the edX team will no longer be supporting new features on the old version of the product.
+This repository has been archived and is no longer supported—use it at your own risk.
+This repository may depend on out-of-date libraries with security issues, and security
+updates will not be provided. Pull requests against this repository will also not be merged.
 
-Note that this repository uses out-of-date versions of libraries that have known security issues. If this repository were to be used, the first step would be to update libraries in the various requirements files to pick up security updates.
+The edX team has officially transitioned all support for the Open Response Assessor (ORA) project to its second iteration (ORA2).
```

Comment:
It's pretty easy to find. If you search the edx organization for "ora" or "ora2", you will immediately find it.